src/views/components/event-timeline/EventTimeline.web.js

```diff
@@ -0,0 +1,31 @@
+import React, { Fragment } from 'react';
+import { Timeline, Event } from 'react-timeline-scribble';
```

Comment:
can you import as ReactTimeline, and change the name of our component to Timeline? does EventTimeline have more accuracy as a descriptor?